org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/logs/LogHelper.java

```diff
@@ -43,27 +43,30 @@ public static void initializeRootLogger(LanguageClient newLanguageClient, LogsSe
 		logger.setUseParentHandlers(false);// Stops output to console
 		
 		// Configure logging LSP client handler
-		if (settings.getClient()) {
-			try {
-				logger.addHandler(LogHelper.getClientHandler(newLanguageClient));
-			} catch (Exception e) {
-				// TODO: handle exception
-			}
+		try {
```

Comment:
I don't understand, if you do that, it means that `xml.logs.client: false` will not work?

Reply:
you're right, I'll fix it

Reply:
Please  rename `LogsSettings#getClient()` to `LogsSettings#isClient()`